config/locales/jobseekers.yml

```diff
@@ -40,6 +40,8 @@ en:
         success: Thank you for your feedback on your job alert
       update:
         success: Your further feedback on job alerts has been sent to the team
+    job_aler_further_feedbacks:
```

Comment:
typo aler

Reply:
Ooo

Reply:
It's also just straight-up wrong lol

Reply:
yeah, have you checked something isnt amis where this translation is used as it was straight up wrong??

Reply:
Fixed now!